### DISCUSSION TITLE
Solr 9 support?

### DISCUSSION THREAD

**Discussion by ilan-prager-privitar** [JanusGraph/janusgraph#3274] [General]
Is there a timeline for upgrade to support Solr 9?
Are there any known changes that are expected to break the existing code in janusgraph-solr?
Thanks!

**Comment by vtslab** [comment]
I do not see any issues or PR's related to solr-9.x. The previous upgrade was made by @farodin91, see https://github.com/JanusGraph/janusgraph/commit/ec6e62d59f9a688ef226da107b1f9cb740807918   There is no general rule what can break when upgrading, it depends on deprecations and library conflicts.

**Comment by ilan-prager-privitar** [comment]
What's would be the proper procedure for opening up an enhancement request for an upgrade like this (is this a GitHub discussion, mailing list, GitHub issue, other?) - thanks!

**Reply by FlorianHockmann** [reply]
You can simply open a GitHub issue for this. Then there's a template for _feature requests_ which is the most appropriate one for dependency updates I think.

**Reply by ilan-prager-privitar** [reply]
Thanks - opened https://github.com/JanusGraph/janusgraph/issues/3277
